test/tools/ossfuzz/yulProto.proto

```diff
@@ -200,6 +205,12 @@ message AssignmentStatement {
   required Expression expr = 2;
 }
 
+message MultiVarAssignmentStatement {
```

Comment:
Why did you split the two?

Reply:
Mainly because of legacy corpus compatibility and simplicity of conversion logic.

Legacy corpus compatibility means we can directly reuse previous fuzzing corpus which contain regular assignments/decls.

Simplicity of conversion logic means that we can cleanly separate conversion of regular and multi var decl/assignment because a regular statement can only use an expression as RHS but a multi var statement must use a function call as RHS.

Reply:
Ok, that makes sense! From a conceptual standpoint, would it be very bad to add a function call node here instead of directly the function id, params, etc?

Reply:
Oh and function params sholud actually be expressions, shouldn't they?

Reply:
You're right. In general, function parameters are expressions. In this PR, they are variable references that could reference a previously declared expression.

So you are suggesting something like
```
let a := foo(add(x0,x1))
```

at the moment, we have
```
let x2 := add(x0,x1)
let a := foo(x2)
```

I would need to think about how to switch to the former from the current PR :)

Reply:
There is a dependency between `FunctionCall` and `FunctionDefinition` messages that is currently sort of "hard coded" in the conversion logic. I would need to think about how to cleanly separate them so that we no longer hard code them.